extensions/wikia/AdEngine/js/slot/bottomLeaderboard.js

```diff
@@ -1,14 +1,27 @@
 /*global define*/
 define('ext.wikia.adEngine.slot.bottomLeaderboard', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.utils.eventDispatcher',
 	'wikia.document',
 	'wikia.domCalculator',
 	'wikia.log',
 	'wikia.throttle',
+	'wikia.viewportObserver',
 	'wikia.window'
-], function (doc, dom, log, throttle, win) {
+], function (
+	adContext,
+	eventDispatcher,
+	doc,
+	dom,
+	log,
+	throttle,
+	viewportObserver,
+	win
+) {
 	'use strict';
 
 	var slotName = 'BOTTOM_LEADERBOARD',
+		breakSlots = ['TOP_RIGHT_BOXAD'],
```

Comment:
What about creating some module that would have some kind of definition of slots that conflicts with each other. In case of some conflict it may update information in the slotContext so slot would be disabled in all features automatically.